src/components/Dex.vue

```diff
@@ -114,7 +213,18 @@ export default {
             }
             this.$store.commit('setCurrentMarket', marketData);
           }
+
+          done(markets);
+        },
+      });
+    },
+
+    loadMarketValuations(markets) {
+      this.$store.dispatch('fetchValuations', {
+        done: (valuations) => {
+          this.$store.commit('appendValuationsToMarktsForPrevewing', valuations);
         },
+        symbols: _.map(markets, 'quoteCurrency'),
```

Comment:
This needs to load `baseCurrency` valuations also, or perhaps only `baseCurrency` valuations, since those are all that are being used.

Reply:
I made it include the `baseCurrency` also.